app/scenes/Settings/Features.tsx

```diff
@@ -56,6 +116,12 @@ function Features() {
           </Trans>
         }
       />
+      <PreferredCollection
```

Comment:
As mentioned in `InputSelect` comment, I think this component can just fetch it's own data rather than having a parent component responsible.